openwisp_controller/config/tests/test_config.py

```diff
@@ -766,7 +781,7 @@ def test_config_modified_sent(self):
 
     def test_check_changes_query(self):
         config = self._create_config(organization=self._get_org())
-        with self.assertNumQueries(1):
+        with self.assertNumQueries(8):
```

Comment:
Can you please track what these additional 7 queries are doing here?
I wonder if there's any inefficiency here, if there's any we have to create an issue to address it.